CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option(WITH_GUI_TESTS "Enable building of GUI tests" OFF)
 option(WITH_DEV_BUILD "Use only for development. Disables/warns about deprecated methods." OFF)
 option(WITH_ASAN "Enable address sanitizer checks (Linux only)" OFF)
 option(WITH_COVERAGE "Use to build with coverage tests (GCC only)." OFF)
+option(WITH_APP_BUNDLE "Enable Application Bundle for macOS" ON)
```

Comment:
Just checked it - most documents/comments in KeepassXC use "OS X". For me it's fine to use the new term or the old term for consistency with the rest of KeepassXC.